masterlist.yaml

```
@@ -15236,11 +15236,19 @@ plugins:
 
   - name: 'Midwood Isle.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/28120/' ]
+    clean:
+      - crc: 0x902458B5
+        util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
```

`util` strings for cleaning data mapped to a `clean` key don't get displayed as Markdown, nor would links be reasonably clickable if they were, so a hyperlink should be avoided in this case.

```suggestion
        util: 'SSEEdit v4.0.4'
```